README.md

```diff
@@ -30,52 +31,122 @@ the branch tracking the latest 0.7 release.
   </a>
 </p>
 
-## [Firezone](https://www.firezone.dev/?utm_source=readme) is a self-hosted VPN server and Linux firewall
+---
 
-- Manage remote access through an intuitive web interface and CLI utility.
-- [Deploy on your own infrastructure](https://docs.firezone.dev/deploy?utm_source=readme)
-  to keep control of your network traffic.
-- Built on [WireGuard®](https://www.wireguard.com/) to be stable, performant,
-  and lightweight.
+**Note**: 🚧 The `main` branch is undergoing major restructuring in preparation
 
-![Firezone Architecture](https://user-images.githubusercontent.com/52545545/183804397-ae81ca4e-6972-41f9-80d4-b431a077119d.png)
+for the release of Firezone 1.x 🚧.
 
-## Get Started
+See the [`legacy` branch](https://github.com/firezone/firezone/tree/legacy) to
+track the latest changes to Firezone 0.7.
 
-Follow our [deploy guide](https://docs.firezone.dev/deploy) to install your
-self-hosted instance of Firezone.
+[Read the 1.0 announcement for more](https://www.firezone.dev/blog/firezone-1-0).
 
-Or, if you're on a
-[supported platform](https://docs.firezone.dev/deploy/docker/supported-platforms?utm_source=readme),
-try our
-[auto-install script](https://docs.firezone.dev/deploy/docker/#option-1-automatic-install).
+---
 
-Using Firezone in production at your organization? Contact us to learn about our
-[Enterprise Plan](https://www.firezone.dev/contact/sales?utm_source=readme).
+## Overview
 
-## Features
+[Firezone](https://www.firezone.dev/?utm_source=readme) is an open source
+platform to securely manage remote access for any-sized organization. Unlike
+most VPNs, Firezone takes a granular, least-privileged approach to access
+management with group-based policies that control access to individual
+applications, entire subnets, and everything in between.
 
+<p align="center">
+  <img width="1439" alt="architecture" src="https://github.com/firezone/firezone/assets/167144/48cd6a1e-2f3f-4ca7-969a-fc5b33e13d1c">
```

Comment:
I would remove this image, bad graphics are worse than no graphics at all, and this one feels a bit hard to follow.